src/backend/egl/display.rs

```diff
@@ -212,6 +201,100 @@ impl EGLDisplay {
         })
     }
 
+    /// Create a new [`EGLDisplay`] from an already initialized EGLDisplay and EGLConfig handle
+    pub unsafe fn from_raw<L>(
+        display: *const c_void,
+        config: *const c_void,
```

Comment:
Guard against null pointers if applicable.

Reply:
I'm wondering if this is the correct thing to even do, would usize or u32 be better?